governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx

```diff
@@ -141,6 +142,9 @@ const Proposal = ({
           })
           proposalIxs.push(parsedInstruction)
         }
+        setIsVerified(
+          !proposalIxs.some((ix) => ix instanceof UnrecognizedProgram)
```

Comment:
I think the definition of verified is more something like :
`ix instanceof PythMultisigInstruction || (ix instanceof WormholeMultisigInstruction && ix.name === "postMessage" && ix.governanceAction instanceOf ExecutePostedVaa && ix.governanceAction.instructions.every(remote_ix => parse(remote_ix) instanceOf PythMultisigInstruction`